Домашнее задание по тестированию API

Eliseev-Max opened the pull request.

Eliseev-Max committed:
Домашнее задание по тестированию API

konflic commented on dog_api/conftest.py:
```diff
@@ -0,0 +1,16 @@
+@pytest.fixture
+def base_url(request):
+    return request.config.getoption("--url")
+
```
Лишняя пустая строка, в конце файла должна быть всего одна, форматтер пайчарма сам сделает правильно если его использовать ;)

konflic commented on jsonplaceholder/conftest.py:
```diff
@@ -0,0 +1,29 @@
+
+@pytest.fixture
+def request_method(request):
+    return getattr(requests, request.config.getoption("--method"))
```
А тут наоборот нет пустой строки.

konflic commented on module/test_module.py:
```diff
@@ -0,0 +1,14 @@
+
+
+def test_check_status_code(verified_url, expected_status_code):
+    """
```
Слишком размашистая докстринга и лишняя пустая строка внутри.

Eliseev-Max replied:
Я руководствовался PEP8, там написано следующее:
"Заметьте, очень важно, чтобы закрывающие кавычки стояли на отдельной строке. А еще лучше, если перед ними будет ещё и пустая строка"
https://pythonworld.ru/osnovy/pep-8-rukovodstvo-po-napisaniyu-koda-na-python.html#id15
Но лучше так не делать?

konflic commented on open_brewery_db/conftest.py:
```diff
@@ -0,0 +1,17 @@
+@pytest.fixture
+def base_url(request):
+    return request.config.getoption("--url")
+
```
Кодстайл.

Eliseev-Max committed:
Правки в conftest.py

Убрал лишнюю пустую строку

Eliseev-Max committed:
Правка кодстайла в jsonplaceholder/conftest.py

Добавил пустую строку в конце кода

Eliseev-Max committed:
Кодстайл

Поправил кодстайл в файле: убрал лишнюю пустую строку между функциями, добавил пустую строку в конце кода

Eliseev-Max reviewed (commented):
Поправил кодстайл, остался один момент спрорный по докстринге.
